packages/starlight/components/SidebarSublist.astro

```diff
@@ -82,6 +87,10 @@ interface Props {
 		display: none;
 	}
 
+	summary > h2 {
+		width: calc(100% - 1.25rem);
+	}
+
```

Comment:
> Tried some flex-shriking on the caret without success.

What issues did you encounter? Disabling shrinking using `flex-shrink: 0;` on the caret icon and using `overflow-wrap: anywhere;` on the title seems to do the trick for me but I may be missing something.